pump/storage/vlog.go

```diff
@@ -339,7 +341,6 @@ func (vlog *valueLog) write(reqs []*request) error {
 			}
 		}
```

Comment:
Should we make sure `fsync` is called whenever `n > 0`?

Reply:
for `vlog.sync = true` case we always make sure call `fsync` before return success, currently, we will return failed if `Write` failed,  so it' s safe to not call `fsync`(we don't return success).
but note it way writes some complete record now but returns failed because we batch multi records in the `vlog.buf.Bytes`.